views/partials/splashscreen.ejs

```diff
@@ -0,0 +1,5 @@
+<div class="splash">
```

Comment:
create a folder in views named layouts and then move the splashscreen.ejs file to this layouts folder, and make other necessary changes accordingly.